opwen_email_client/webapp/tasks.py

```diff
@@ -36,16 +39,28 @@ def sync(*args, **kwargs):
         user_store=webapp.ioc.user_store,
     )
 
-    start_internet_connection = StartInternetConnection(
-        state_dir=AppConfig.STATE_BASEDIR,
-        modem_config_dir=AppConfig.MODEM_CONFIG_DIR,
-        sim_config_dir=AppConfig.SIM_CONFIG_DIR,
-        sim_type=AppConfig.SIM_TYPE,
-    )
+    def check_connection():
```

Comment:
Could you make this function take a hostname and port as arguments and then move it to `opwen_email_client/util/network.py`? It's generic and re-usable so no need to hide it inside of the Celery tasks module.

Reply:
Ok will do!